### PR TITLE
SISRP-46661 sir accept button

### PR DESCRIPTION
**Pull request by frankiepun**
https://jira-secure.berkeley.edu/browse/SISRP-46661

### CODE REVIEW

**Comment by redconfetti** [comment]
@frankiepun - You should link this pull request to the JIRA it's related to

File: src/assets/templates/widgets/sir/sir_item.html [hunk 1]
```diff
@@ -138,9 +138,9 @@ <h3>Next Steps</h3>
           </div>
           <div class="cc-widget-sir-button-container">
             <button
-              class="cc-button cc-button-blue"
-              type="submit"
-              data-ng-disabled="!api.user.profile.actAsOptions.canPost || !sirItem.isFormValid || sirItem.isSubmitting"
+              ng-class="setButtonClass()"
+              data-ng-disabled="!api.user.profile.actAsOptions.canPost || !sirItem.form.option || sirItem.isSubmitting"
```
Comment (redconfetti) [inline src/assets/templates/widgets/sir/sir_item.html]: @frankiepun - I wonder if this logic can be shared in the controller in a function called `formDisabled()` and used with `$scope.setButtonClass` also... or is the logic much different between the two?

File: src/assets/stylesheets/_buttons.scss [hunk 2]
```diff
@@ -125,6 +125,23 @@ button {
...
+  border: 1px solid transparent;
+  color: $cc-color-white;
+  &:focus, &:hover {
+    background: $cc-color-bali-hai !important;
+    color: $cc-color-white;
+  }
+  &.cc-button-large {
+    background: linear-gradient($cc-color-bali-hai, $cc-color-lochmara), $cc-color-lochmara;
+    &:focus, &:hover {
+      background: linear-gradient($cc-color-bali-hai, $cc-color-blue-lagoon);
```
Comment (marcpechaitis) [inline src/assets/stylesheets/_buttons.scss]: Are we using gradients on our react buttons?

cc: @developish, @redconfetti, @frankiepun, @nkutub

File: src/assets/templates/widgets/sir/sir_item.html (hunk 1 above)
Comment (frankiepun) [reply src/assets/templates/widgets/sir/sir_item.html]: Thanks Jason. I always forgot to add the PR to the ticket. I have just added it.

Excellent point of sharing the logic! The user has a special requirement of two button-states: a normal disabled, non-clickable button and a clickable greyed-out button. data-ng-disabled handles the normal disabled button case. The clickable button has one more condition in setButtonClass(). Let me try to consolidate the logic.

**Pull request merged**